bindata/network/openshift-sdn/sdn-ovs.yaml

```diff
@@ -115,9 +115,13 @@ spec:
             if [[ -f /var/run/openvswitch/flows.sh ]]; then
               echo "$(date -u "+%Y-%m-%d %H:%M:%S") info: Adding br0 if it doesn't exist ..." 2>&1
               /usr/bin/ovs-vsctl --may-exist add-br br0 -- set Bridge br0 fail_mode=secure protocols=OpenFlow13
+              echo "$(date -u "+%Y-%m-%d %H:%M:%S") info: Renaming recovery file in case we fail during restoration ..." 2>&1
```

Comment:
we don't need to log every single step. Just do the `mv` and `sh -x` both after the "adding flows" message